spec/folder_spec.rb

```diff
@@ -165,12 +168,13 @@ def post_create_folder path, params={}
       expect(json_body[:error]).to eq('Cannot overwrite existing file')
     end
 
-    it 'refuses to create folders with non-existent parent folder' do
+    it 'allows creation of folders with non-existent parent folder' do
       token_header(:editor)
       post_create_folder('blueprints/Helmet Blueprints')
 
-      expect(last_response.status).to eq(422)
-      expect(json_body[:error]).to eq('Invalid folder: "blueprints"')
+      expect(last_response.status).to eq(200)
+      folder = Metis::Folder.last
+      expect(folder.folder_path).to eq([ 'blueprints', 'Helmet Blueprints'])
```

Comment:
Although implicit in folder_path, perhaps validate that Metis::Folder.count == 2 here.